phpunit.xml.dist

```diff
@@ -8,6 +8,7 @@
         beStrictAboutTodoAnnotatedTests="true"
         failOnWarning="true"
         failOnRisky="true"
+        stopOnRisky="true"
```

Comment:
this should be reverted as well

Reply:
OK, I added this line, because CI not showed deprecated warnings on PHP 8.4 CI run